README.md

```diff
@@ -130,11 +130,10 @@ build.
 
 MSVC builds of Rust additionally require an installation of Visual Studio 2017
 (or later) so `rustc` can use its linker.  The simplest way is to get the
-[Visual Studio Build Tools] and check the “C++ build tools” workload.
+[Visual Studio], check the “C++ build tools” and “Windows 10 SDK” workload.
 
-Windows 10 SDK is also required now. 
 
-[Visual Studio Build Tools]: https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019
+[Visual Studio]: https://visualstudio.microsoft.com/zh-hans/downloads/
```

Comment:
The old link is redirected to vs 2019, and there is not a standalone C++ build tools.

Reply:
https://visualstudio.microsoft.com/downloads/ is language neutral link.

Reply:
I shouldn't paste that url.